hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/state/HdfsStore.java

```diff
@@ -0,0 +1,21 @@
+package org.apache.mesos.hdfs.state;
```

Comment:
Ok, I think your point makes sense. Can we name this with the naming convention of IHdfsStore?